build/packages/rpm/server/SPECS/victoriametrics-gateway.spec

```diff
@@ -0,0 +1,41 @@
+%undefine _missing_build_ids_terminate_build
+
+%define copying() \
+%if 0%{?fedora} >= 21 || 0%{?rhel} >= 7 \
+%license %{*} \
+%else \
+%doc %{*} \
+%endif
+
+%global repo            VictoriaMetrics
+%global provider        github.com/VictoriaMetrics/%{repo}
+%global commit          v1.82.1
+
+Name:           percona-victoriametrics-gateway
+Version:        1.82.1
+Release:        1%{?dist}
+Summary:        VictoriaMetrics gateway solution
+License:        Apache-2.0
+URL:            https://%{provider}
+Source0:        https://%{provider}/releases/download/%{commit}/vmutils-linux-amd64-%{commit}-enterprise.tar.gz
+
+
+%description
+%{summary}
+
+
+%prep
+%setup -q -c
+
+
+%install
+install -D -p -m 0755 ./vmgateway-prod %{buildroot}%{_sbindir}/vmgateway
+
+
+%files
+%{_sbindir}/vmgateway
+
+
+%changelog
+* Mon Oct 24 2022 Michal Kralik <michal.kralik@percona.com> - 1.82.1
```

Comment:
```suggestion
* Mon Oct 24 2022 Michal Kralik <michal.kralik@percona.com> - 1.82.1-1
```

Reply:
What is the reason we would want to include the `-1` suffix?
I tried to stay consistent with the existing `percona-victoriametrics` rpm where we don't have suffixes yet.

Reply:
In fact, we do have suffixes everywhere if you look at this:
`[root@eda3600eace8 ansible]# rpm -qa | grep percona
percona-victoriametrics-1.77.1-1.el7.x86_64
percona-postgresql14-14.5-3.el7.x86_64
percona-postgresql11-contrib-11.17-3.el7.x86_64
percona-postgresql14-server-14.5-3.el7.x86_64
percona-grafana-8.3.10-96.2209091136.2680b34.el7.x86_64
percona-alertmanager-0.22.0-3.el7.x86_64
percona-qan-api2-2.31.0-16.2209120831.500663d.el7.x86_64
percona-postgresql11-libs-11.17-3.el7.x86_64
percona-postgresql14-libs-14.5-3.el7.x86_64
percona-postgresql11-11.17-3.el7.x86_64
percona-postgresql14-contrib-14.5-3.el7.x86_64
percona-postgresql11-server-11.17-3.el7.x86_64
percona-release-1.0-27.noarch
percona-dashboards-2.31.0-19.2209151640.25fba72.el7.x86_64`